internal/clients/init.go

```diff
@@ -182,56 +182,74 @@ func checkConsulAvailable() bool {
 
 func initializeClients() {
 	isRegistry := common.UseRegistry
-	var waitGroup sync.WaitGroup
+	var waitGroup *sync.WaitGroup
```

Comment:
WaitGroup is nil

panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x47ce3d]

goroutine 22 [running]:
sync.(*WaitGroup).state(...)
	D:/Go/src/sync/waitgroup.go:33
sync.(*WaitGroup).Add(0x0, 0x8)
	D:/Go/src/sync/waitgroup.go:54 +0x2d
github.com/edgexfoundry/device-sdk-go/internal/clients.initializeClients()
	D:/Development/EdgeXFoundry/device-sdk-go/internal/clients/init.go:186 +0x5a
github.com/edgexfoundry/device-sdk-go/internal/clients.InitDependencyClients(0x0, 0x0)
	D:/Development/EdgeXFoundry/device-sdk-go/internal/clients/init.go:46 +0x70
github.com/edgexfoundry/device-sdk-go.(*Service).Start(0xc0000e0600, 0xc0000e0660, 0x0, 0x0)
	D:/Development/EdgeXFoundry/device-sdk-go/service.go:72 +0x3b
created by github.com/edgexfoundry/device-sdk-go/pkg/startup.startService
	D:/Development/EdgeXFoundry/device-sdk-go/pkg/startup/bootstrap.go:54 +0x1a8